roseau/load_flow/utils/mixins.py

```diff
@@ -26,7 +32,103 @@ def from_dict(cls, data: JsonDict, *args: Any) -> Self:
         """Create an element from a dictionary."""
         raise NotImplementedError
 
+    @classmethod
+    def from_json(cls, path: StrPath) -> Self:
+        """Construct an electrical network from a json file created with :meth:`to_json`.
+
+        Args:
+            path:
+                The path to the network data file.
+
+        Returns:
+            The constructed network.
+        """
+        data = json.loads(Path(path).read_text())
+        return cls.from_dict(data=data)
+
     @abstractmethod
     def to_dict(self) -> JsonDict:
         """Return the element information as a dictionary format."""
         raise NotImplementedError
+
+    def to_json(self, path: StrPath) -> Path:
+        """Save the current network to a json file.
+
+        .. note::
```

Comment:
Why is this method here and not overridden in `ElectricalNetwork` ? To be called from any element ?

Reply:
Yes. If you want to have a library of loads, lines, etc. You can export elements to files.